plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/MethodConverter.kt

```diff
@@ -39,6 +39,13 @@ class MethodConverter(
         return result
     }
 
+    override fun addLoopIdentifier(labelName: String, index: Int) {
+        propertyResolver = propertyResolver.addLoopIdentifier(labelName, index)
```

Comment:
Should we not lose this identifier once the loop is over?

Reply:
I think that when the loop is over, a scope switch happens and the identifier is lost for free